service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java

```diff
@@ -232,22 +250,27 @@ private Set<String> getRepresentativesEmails(CaseData caseData,
     @Async
     public void notifyEmailServedRepresentatives(final AdditionalApplicationsUploadedEvent event) {
         final CaseData caseData = event.getCaseData();
-        NotifyData notifyData = contentProvider.getNotifyData(caseData);
 
-        Set<String> emailRepresentatives = getRepresentativesEmails(caseData, EMAIL);
+        AdditionalApplicationsBundle bundleUploaded = caseData.getAdditionalApplicationsBundle().get(0).getValue();
 
-        if (!emailRepresentatives.isEmpty()) {
-            representativeNotificationService.sendNotificationToRepresentatives(
-                caseData.getId(),
-                notifyData,
-                emailRepresentatives,
-                INTERLOCUTORY_UPLOAD_NOTIFICATION_TEMPLATE_PARTIES_AND_OTHERS
-            );
+        if (bundleUploaded.getC2DocumentBundle() != null || bundleUploaded.getOtherApplicationsBundle() != null) {
```

Comment:
same comment as the one on method `notifyDigitalRepresentatives`

Reply:
moved the retrieving and checking to separate method